setup.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+# SPDX-FileCopyrightText: © 2020 Open Networking Foundation <support@opennetworking.org>
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import absolute_import
```

Comment:
This can be dropped since we're only supporting Python3.7+ right?

Reply:
Correct, but `pylint --py3k` will complain if it's not there.  That's probably configurable.

Reply:
Ah I see, we use `flake8` for linting so I'm not as familiar with `pylint` :)